docs/analytics/resources/emr-vars.yml

```diff
@@ -5,7 +5,7 @@ region: 'us-east-1'
 instance_groups:
   master:
     num_instances: 1
-    type: m4.medium
+    type: m4.large
```

Comment:
It's because of the hadoop 2 upgrade, unfortunately.  ref [edX Analytics Team's post on EMR 4.x, section 2.3](https://groups.google.com/forum/#!msg/openedx-ops/eCyNwYdhVcU/cOIHPLZiBwAJ).